src/app/shared/editor/components/base-editor/base-editor.component.ts

```diff
@@ -10,7 +10,7 @@ import { EditorError } from "../../models/error.model";
 export abstract class BaseEditorComponent implements OnChanges {
     @Input({ required: true }) public template = "";
     @Input() public error: MaybeNull<string>;
-    @Input() public height: MaybeUndefined<string>;
+    @Input() public height: MaybeUndefined<string> = "220px";
```

Comment:
Do we need undefined if we have a default?

Reply:
Yes, it is unnecessary